plugins/cache-ehcache/src/main/java/org/opensearch/cache/store/disk/EhcacheDiskCache.java

```diff
@@ -113,7 +116,7 @@ public class EhcacheDiskCache<K, V> implements ICache<K, V> {
     private final Class<K> keyType;
     private final Class<V> valueType;
     private final TimeValue expireAfterAccess;
-    private final CacheStatsHolder cacheStatsHolder;
+    private final CacheStatsHolderInterface cacheStatsHolder;
```

Comment:
Explicitly appending `Interface` doesn't seem a good practice. We should keep the interface as `CacheStatsHolder` and change the concrete class name instead.
So below CacheStatsHolder concrete class should be renamed to `DefaultCacheStatsHolder`.